flyteadmin/auth/authzserver/token_test.go

```diff
@@ -73,7 +73,7 @@ func Test_tokenEndpoint(t *testing.T) {
 
 		m := map[string]interface{}{}
 		assert.NoError(t, json.Unmarshal(rw.Body.Bytes(), &m))
-		assert.Equal(t, 0.5*time.Hour.Seconds()-1, m["expires_in"])
+		assert.GreaterOrEqual(t, m["expires_in"], 0.5*time.Hour.Seconds()-5)
```

Comment:
Personally I would separate this using a dedicated fix but not sure if flyte guys care about